packages/go_router_builder/example/pubspec.yaml

```diff
@@ -18,15 +17,8 @@ dev_dependencies:
   build_verify: ^3.0.0
   flutter_test:
     sdk: flutter
-  # Uses path dependency override
-  go_router_builder:
+  go_router_builder: ^1.0.0
   test: ^1.17.0
 
 flutter:
   uses-material-design: true
-
-dependency_overrides:
-  go_router:
-    path: ../../go_router
-  go_router_builder:
```

Comment:
Don't we WANT to leave the path-based dep for the package hosting the example?

Reply:
That's my preference too. My understanding is that the `make-deps-path-based` script modifies the pubspec, which ensures it's tested even if there are no changes to it.

I'll put this back and make sure everything still works...

Reply:
It looks like dependency_overrides isn't supported by `make-deps-path-based`, so I opted to use the path-based dependency directly

Reply:
Hrm. The issue then is that the "at my desk" experience here is ...weird. I guess weirdness will be caught in CI 🤷

Reply:
Why is that?

Reply:
If you change go_router_builder locally...fixing things. You may make a change that'd break the example – but you won't know running the example code locally (unless you remember to change the path dep) until you go to CI. 🤷

Reply:
I made the example path-based in https://github.com/flutter/packages/pull/2252/commits/b32413218e8cda717fecf9b915c76671b4d37e4b so I think any changes to go_router_builder will show up in the example - sorry for the churn on this pull request.

Reply:
Had to change this back, see my comment below.